tests/asgi/tests.py

```diff
@@ -315,6 +316,7 @@ class SignalHandler:
 
             threads = []
 
+            @async_unsafe
```

Comment:
Commenting this doesn't cause a failure.

Reply:
Changing the call `asend` to `send` does produce failures independently of whether this decorator is present or not. Are you certain is needed?

Reply:
Thanks - I have gone over this and confirmed its probably not needed. Since its the only change here I'm happy to remove. There is a failure if request.close() is called without the sync_to_async wrapper, but its not necessary to have this decorator as the thread tests also fail in that case.